images/nvidia-device-plugin/tests/README.md

```diff
@@ -0,0 +1,135 @@
+# Notes on how this image was tested
+
+The `nvidia-device-plugin` image required GPUs to test, therefore it is not currently covered by automated tests.
+
+This document describes the manual steps taken to test the image on GCP.
+
+Note, for testing we use a the `nvidia/gpu-operator` helm chart which requires bash so you will want to use the `latest-dev` variant.
```

Comment:
another option is to override the daemonset to call `nvidia-device-plugin` directly

i tested it and the `/bin/entrypoint.sh` provides extra checks but isnt needed really.

Reply:
Yeah - I was thinking for now to keep the steps as straight forward as possible as there's a lot going on there.  If you want I can add this in but using `latest-dev` seems like a fair option, also for testing folks might like bash to exec and noodle around the running pod.

Reply:
sgtm

Reply:
cool thanks 👍